cloudformation.yml

```diff
@@ -73,6 +73,34 @@ Resources:
                       Resource:
                             - !Sub arn:aws:dynamodb:*:${AWS::AccountId}:table/floodgate-*-${Stage}
 
+    Ec2DescribeInstancesPolicy:
+        Type: AWS::IAM::Policy
+        Properties:
+            PolicyName: ec2-describe-instances
+            PolicyDocument:
+                Statement:
+                    - Effect: Allow
+                      Action:
+                          - ec2:DescribeInstances
+                          - ec2:DescribeTags
+                      Resource: "*"
+            Roles:
+                - !Ref: RootRole
```

Comment:
Same as https://github.com/guardian/itunes-rss/pull/87#discussion_r635161155 - otherwise looks fine. 

With the amiEncrypted param, I believe AMIgo relies on a lambda in the CAPI account to subscribe to an SNS topic in the Deploy Tools account, which then copies an encrypted copy over to the CAPI AWS account. I presume the pre-existing lambdas have the right subscriptions to make this work for floodgate in PROD (there aren't many CAPI subscriptions in CODE but Floodgate has no CODE environment).